lib/database/models/lhcPeriodsStatistics.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+const Sequelize = require('sequelize');
+
+module.exports = (sequelize) => {
+    const LhcPeriodStatistics = sequelize.define(
+        'LhcPeriodForPhysicsRuns',
+        {
+            id: {
+                allowNull: false,
+                autoIncrement: true,
+                primaryKey: true,
+                type: Sequelize.INTEGER,
+            },
+            name: {
+                type: Sequelize.STRING,
+                allowNull: false,
+            },
+            avgEnergy: {
+                type: Sequelize.FLOAT,
+                allowNull: true,
+                field: 'avgEnergy',
+            },
+        },
+        { tableName: 'lhc_periods_with_statistics' },
+    );
+
+    LhcPeriodStatistics.associate = (models) => {
+        LhcPeriodStatistics.hasMany(models.Run, { foreignKey: 'lhcPeriodId', as: 'lhcPeriodsStatistics' });
```

Comment:
I'm not sure about it, I would rather put the relation on the period (which in turn has relation to runs)

Reply:
done